src/components/Tooltip.tsx

```diff
@@ -25,7 +25,7 @@ export const Tooltip = styled(ReactTooltip).attrs({
     font-size: 12px;
     line-height: 16px;
     letter-spacing: -0.2px;
-    max-width: 300px;
+    max-width: 100%;
```

Comment:
that will affect other components, we can redeclare that with a costume style in article-editor

Reply:
I've reviewed the other components, and they weren't affected. Setting `max-width: 100%` would be ideal, but I’ve added a custom solution for now  https://github.com/Atypon-OpenSource/manuscripts-article-editor/pull/543/commits/fd2c7c440da21091f766e45e5161a7eb7fb0bdb9. I’ll go ahead and close this PR.